src/lib/chat/matrix/chat-message.ts

```diff
@@ -46,9 +48,35 @@ export async function mapMatrixMessage(matrixMessage, sdkMatrixClient: SDKMatrix
     },
     isAdmin: false,
     optimisticId: content.optimisticId,
-    ...{ mentionedUsers: [], hidePreview: false, media: null, image: null, admin: {} },
+    ...{
+      admin: {},
+      mentionedUsers: [],
+      hidePreview: false,
+      media: null,
+      image: null,
+    },
     parentMessageText: '',
     parentMessageId: parent ? parent['m.in_reply_to']?.event_id : null,
     ...(await parseMediaData(matrixMessage)),
   };
 }
+
+export function mapEventToAdminMessage(matrixMessage) {
+  const { event_id, content, origin_server_ts, sender: senderId, type } = matrixMessage;
+
+  const adminMessageType =
+    type === CustomEventType.USER_JOINED_INVITER_ON_ZERO
+      ? AdminMessageType.JOINED_ZERO
+      : AdminMessageType.CONVERSATION_STARTED;
+
+  return {
+    id: event_id,
+    message: 'Conversation was started',
+    createdAt: origin_server_ts,
+    isAdmin: true,
+    sender: {
+      userId: senderId,
```

Comment:
Do we have to put a sender here? I don't think anyone is the sender of these "messages".

Reply:
Correct yeah no one is the sender but if I do remove this 

<img width="485" alt="Screenshot 2023-11-13 at 17 09 31" src="https://github.com/zer0-os/zOS/assets/39112648/f04c54e2-f415-49e1-b12a-3b6d28993a3d">

Suggesting this would need to be handled in mapChannelMessages. Shall take a look in the follow ups